README.md

```diff
@@ -4,14 +4,49 @@
 
 ![Build Status](https://github.com/Shopify/vscode-ruby-lsp/workflows/CI/badge.svg)
 
-# vscode-ruby-lsp
```

Comment:
I prefer to keep the old name. `Ruby LSP` is same as the actual `ruby-lsp`'s, which is confusing.

Or let's make it `Ruby LSP (VS Code extension)`?